.zenodo.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
This file is used to auto-generate DOI's at zenodo.org.  Let me know if any of the creators information needs updating.

Reply:
I've updated my affiliation.